tests/Serializer/AbstractSerializerTest.php

```diff
@@ -317,20 +317,6 @@ public function testLongString(bool $serializeAllObjects): void
         }
     }
 
-    public function testLongStringWithOverwrittenMessageLength(): void
```

Comment:
Why did you move those?

Reply:
Because the tests didn't run.

Reply:
Wasn't it possible to just add the `$options` parameter to the function?